.github/workflows/pull-image-builder-test.yml

```diff
@@ -0,0 +1,18 @@
+name: pull-image-builder-test
+
+on:
+  pull_request_target:
+    types: [opened, edited, synchronize, reopened, ready_for_review]
+    paths:
+      - ".github/workflows/pull-image-builder-test.yml"
+      - ".github/workflows/image-builder.yml"
+      - ".github/actions/expose-jwt-action/**"
+      - ".github/actions/image-builder/**"
+
+jobs:
+  test_image_builder:
+    uses: ./.github/workflows/image-builder.yml # Usage: kyma-project/test-infra/.github/workflows/image-builder.yml@main
+    with:
+      name: test-infra/ginkgo
+      dockerfile: prow/images/ginkgo/Dockerfile
```

Comment:
does it work without providing name of the env file? 
https://github.com/kyma-project/test-infra/blob/main/prow/images/ginkgo/envs

Reply:
It should, but I added env file support